CONTRIBUTING.md

```diff
@@ -1,4 +1,42 @@
-# Contributing to plotly.js
+# Contributing to Plotly.js
+
+Thanks for your interest in contributing to Plotly.js! We are actively looking for
+diverse contributors, with diverse background and skills.
+
```

Comment:
put here a few sentences explaining the organization of this document? It is quite long so it might be worth it. In particular some people will be looking specifically for out to set up a development environment.